src/Uno.UI.Wasm/ts/Windows/Devices/Midi/Internal/WasmMidiAccess.ts

```diff
@@ -0,0 +1,37 @@
+﻿namespace Uno.Devices.Midi.Internal {
+	export class WasmMidiAccess {
+
+
+		private static midiAccess: WebMidi.MIDIAccess;
+
+		private static dispatchRequest: (hasAccess: boolean) => number;
+
+		private static initialize() {			
+			if (!this.dispatchRequest) {
+				this.dispatchRequest = (<any>Module).mono_bind_static_method("[Uno] Uno.Devices.Midi.Internal.WasmMidiAccess:DispatchRequest");
+			}
+		}
+
+		public static request() {
+			WasmMidiAccess.initialize();
+			if (navigator.requestMIDIAccess) {
+				console.log('This browser supports WebMIDI!');
+				navigator.requestMIDIAccess()
+					.then(
+						(midi: WebMidi.MIDIAccess) => {
+							WasmMidiAccess.midiAccess = midi;
+							return WasmMidiAccess.dispatchRequest(true);
+						},
+						() => WasmMidiAccess.dispatchRequest(false));
+			}
+			else {
+				console.log('WebMIDI is not supported in this browser.');
+				WasmMidiAccess.dispatchRequest(false);
+			}
+		}
+
+		public static getMidi(): WebMidi.MIDIAccess {
+			return this.midiAccess;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)